proc/proc_test.go

```diff
@@ -2374,8 +2373,25 @@ func TestIssue683(t *testing.T) {
 				break
 			}
 		}
-		if !goterror {
-			t.Fatal("expeceted an error we didn't get")
+		ver, _ := ParseVersionString(runtime.Version())
+
+		if !(ver.Major < 0 || ver.AfterOrEqual(GoVersion{1, 8, -1, 0, 0})) {
```

Comment:
Will do in a separate PR since it doesn't have anything to do with 1.8 compatibility